frontend/src/components/Contexts/MissionRunsContext.tsx

```diff
@@ -4,14 +4,13 @@ import { BackendAPICaller } from 'api/ApiCaller'
 import { SignalREventLabels, useSignalRContext } from './SignalRContext'
 import { translateSignalRMission } from 'utils/EnumTranslations'
 
-const upsertList = (list: Mission[], mission: Mission) => {
+const upsertMissionList = (list: Mission[], mission: Mission) => {
     let newList = [...list]
     const i = newList.findIndex((e) => e.id === mission.id)
     if (i > -1) newList[i] = mission
     else newList.push(mission)
     return newList
```

Comment:
how do you feel about calling this newMissionList to keep the same line of thought?